<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>350</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>350</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MHelper</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
background-color: rgb(255, 255, 255);
}

QPushButton{
background-color: rgb(136, 216, 255);
border-radius: 25px;
}

QPushButton:hover{
background-color: rgb(136, 206, 235);
border-radius: 25px;
}

QPushButton:pressed{
background-color: rgb(136, 206, 235);
border-radius: 20px;
}

QLabel{
 font: 30pt/30pt fantasy;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="name_lbl">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>40</y>
      <width>191</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>MHelper</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addmail_btn">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>160</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить письмо</string>
    </property>
   </widget>
   <widget class="QPushButton" name="seemail_btn">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>220</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Мои письма</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
